core/server/worker/src/main/java/alluxio/worker/netty/DataServerBlockReadHandler.java

```diff
@@ -170,13 +170,15 @@ private void openBlock(Channel channel) throws Exception {
       return;
     }
 
-    int retryInterval = Constants.SECOND_MS;
+    int retryInterval = Constants.SECOND_MS * 2;
```

Comment:
Why is this interval increased, is it sensitive to the environment or will 2 seconds be good in all cases?